internal/cmd/ship/cmd.go

```diff
@@ -169,6 +170,35 @@ Start:
 		}
 		shipProgramSquashMerge(prog, repo, sharedData, squashMergeData, message)
 	}
+	if sharedData.config.NormalConfig.ProposalsShowLineage == forgedomain.ProposalsShowLineageCLI {
+		if connector, hasConnector := sharedData.connector.Get(); hasConnector {
+			if proposalFinder, hasProposalFinder := connector.(forgedomain.ProposalFinder); hasProposalFinder {
+				tree, err := forge.NewProposalStackLineageTree(forge.ProposalStackLineageArgs{
+					Connector:                proposalFinder,
+					CurrentBranch:            sharedData.initialBranch,
+					Lineage:                  sharedData.config.NormalConfig.Lineage,
+					MainAndPerennialBranches: sharedData.config.MainAndPerennials(),
+				})
+				if err != nil {
+					fmt.Printf("failed to update proposal stack lineage: %s\n", err.Error())
```

Comment:
This error message seems wrong. forge.NewProposalStackLineageTree doesn't update the proposals.